app/forms/nominate_induction_tutor_form.rb

```diff
@@ -19,9 +19,7 @@ def school
   end
 
   def email_already_taken?
-    User.left_outer_joins(:induction_coordinator_profile)
-        .where(induction_coordinator_profile: { id: nil })
-        .exists?(email: email)
+    ParticipantProfile.ecf.joins(:user).where(user: { email: email }).any?
```

Comment:
This theoretically means we could end up with someone who is both and admin/lead provider and a coordinator. I don't think it will ever happen, but something to consider

Reply:
Yes, I know, not sure if it's something to really worry about?
I think ideally we'd have all the profiles inheriting from some common base which would simplify things.